README.md

```diff
@@ -15,6 +15,43 @@ This project consists of the following modules:
 * [Android Studio](https://developer.android.com/studio/index.html), Provides the fastest tools for building apps on every type of Android device.
 * [Gradle](https://gradle.org/), An open source build automation system.
 
+## Testing and CI
+
+### Circle
+By default this boilerplate comes configured to run on [CircleCI](circleci.com). The YML file `circle.yml` will setup the dependencies, run the tests configured in `scripts/ci.sh` and export relevant reports and artefacts.
+
+If you have configured the environment variables defined in the "Firebase cloud testing" the tests will be run on the [Firebase test lab](https://firebase.google.com/docs/test-lab/) otherwise your tests will be run on an emulator.
+
+### Firebase cloud testing
+This boilerplate ships with Firebase cloud testing built in. You simply need to enter the following environment variables to make it work:
+
+- `GCLOUD_SERVICE_ACCOUNT` - The service account email address is listed in the Firebase console here: https://console.firebase.google.com/project/<projectId>/settings/serviceaccounts/adminsdk
+- `GCLOUD_SERVICE_KEY` - This is an export of the contents of the JSON secret file that can be obtained by generating a new private key here: https://console.firebase.google.com/project/<projectId>/settings/serviceaccounts/adminsdk
```

Comment:
Add backticks surrounding `https://console.firebase.google.com/project/<projectId>/settings/serviceaccounts/adminsdk` to let markdown parse the placeholder correctly.